wgengine/userspace.go

```diff
@@ -412,55 +406,13 @@ func (e *userspaceEngine) Reconfig(cfg *wgcfg.Config, dnsDomains []string) error
 func (e *userspaceEngine) GetFilter() *filter.Filter {
 	e.mu.Lock()
```

Comment:
Is e.tundev guarded by e.mu? If so, it's in the wrong paragraph of fields in userspaceEngine. If not, why is the lock held here?